scripts/items/sheet_of_divine_tunes.lua

```diff
@@ -0,0 +1,18 @@
+-----------------------------------
+-- ID: 6704
+-- Item: Divine Tunes
+-- Grants keyitem: SHEET_OF_DIVINE_TUNES
+-----------------------------------
+---@type TItem
+local itemObject = {}
+
+itemObject.onItemCheck = function(target, item, param, caster)
+    -- TODO: The error message returned by this is wrong
+    return target:hasKeyItem(xi.keyItem.SHEET_OF_DIVINE_TUNES) and 1 or 0
+end
+
+itemObject.onItemUse = function(target)
+    npcUtil.giveKeyItem(target, xi.keyItem.SHEET_OF_DIVINE_TUNES)
```

Comment:
P.S. this probably won't produce the right messaging of retail, so we need a TODO to cap it

See that left arrow? the normal npcUtil.giveKeyItem doesn't generate this message
![image](https://github.com/user-attachments/assets/93b7924d-672e-45ec-8c42-f0ec95140410)

Reply:
Ohh quite right